pkgs/os-specific/linux/nvidia-x11/generic.nix

```diff
@@ -23,35 +23,42 @@
 with stdenv.lib;
 
 assert (!libsOnly) -> kernel != null;
+assert (versionOlder version "391") -> sha256_32bit != null;
+assert !(versionOlder version "391") -> stdenv.hostPlatform.system == "x86_64-linux";
```

Comment:
Some nitpicking: All these parens up to line 25 can be removed, Nix doesn't need them

Reply:
If I'm interpreting this correctly, it means remove lines 26, 27?

Reply:
No, just remove the parens:

```
assert versionOlder version "391" -> sha256_32bit != null;
assert ! versionOlder version "391" -> stdenv.hostPlatform.system == "x86_64-linux";
```nix